### PR TITLE
Refactor uwu immunity

### PR DESCRIPTION
**Pull request by Asterisk007**
Relies on a list of usernames stored in the local JSON database, instead of user roles. Obsoletes the `uwu-immune` role.

### CODE REVIEW

**Comment by CodeMastr3** [comment]
Sadly because I don't know too much about json files or the small database the bot has I don't feel qualified to accept this pull request

File: bot.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+from click import pass_context
```
Comment (Ramsfield) [inline bot.py:1]: What's the purpose of this library? Where is it used?

File: bot.py [hunk 2]
```diff
@@ -207,22 +239,22 @@ async def uwu(ctx, arg1=""):
 
         if(not argIsText):
             message = await channel.fetch_message(msg_id)
-            if "uwu-immune" in [r.name.lower() for r in message.author.roles]:
+            if message.author.name + message.author.discriminator in immuwune:
                 await ctx.send("UwU this usew is uwu-immune! sowwy... 😭")
```
Comment (Ramsfield) [inline bot.py:243]: Should we make it say "immuwune"? I think it'd be funy

File: bot.py (hunk 1 above)
Comment (Asterisk007) [reply bot.py:1]: That looks to be errata as a result of VSCode autoimporting something based on a typo. I'll remove it.

File: bot.py [hunk 3]
```diff
@@ -207,22 +239,22 @@ async def uwu(ctx, arg1=""):
...
                 return
             else:
                 message = message.content
         else:
             # arg1 is original text that wants to be uwu-ized
-            if "uwu-immune" in [r.name.lower() for r in ctx.message.author.roles]:
+            if ctx.message.author.name + ctx.message.author.discriminator in immuwune:
                 await ctx.send("UwU you'we immune to the uwu! sowwy... 😭")
```
Comment (Ramsfield) [inline bot.py:250]: Here as well

Comment (Asterisk007) [reply bot.py:249]: 🤔I mean....I guess. Especially with this new word I've coined.

File: bot.py (hunk 1 above)
Comment (Ramsfield) [reply bot.py]: Appreciated <3

File: bot.py (hunk 3 above)
Comment (Ramsfield) [reply bot.py]: Your call, it doesn't affect the PR.

Comment (Asterisk007) [reply bot.py]: Well I did it anyway so :P

**Pull request merged**